src/test/java/org/isf/security/TokenProviderTest.java

```diff
@@ -22,25 +22,37 @@
 package org.isf.security;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertThrows;
```

Comment:
```suggestion
import static org.junit.jupiter.api.Assertions.assertThrows;
```

Reply:
Why use junit 4 code when everything else is junit 5+ ?

Reply:
automatic import... fixed, thanks!